config/config.go

```diff
@@ -139,6 +139,12 @@ type BaseConfig struct {
 
 	// NoMainOverride forces the "nomain" builds to run on the mainnet
 	NoMainOverride bool `mapstructure:"no-main-override"`
+
+	// ATX version switches
+	// Each entry states on which publish epoch given ATX version becomes valid.
+	// Note: There is always one valid version at any given time.
+	// ATX V1 starts with epoch 0 unless configured otherwise.
+	AtxVersions map[types.EpochID]types.AtxVersion `mapstructure:"atx-versions"`
```

Comment:
I feel like the validation should already happen when parsing the config, similar how we do it with `PostRandomXMode`, `PostProviderID`, etc.

Then the handler doesn't need to contain the validation logic.